clients/spark/core/src/main/scala/io/treeverse/clients/StorageUtils.scala

```diff
@@ -0,0 +1,37 @@
+package io.treeverse.clients
+
+import java.net.URI
+
+object StorageUtils {
+  val StorageTypeS3 = "s3"
+  val StorageTypeAzure = "azure"
+  val SchemeDelimiter = "://"
```

Comment:
I think this last constant is a bit ugly.  It will never change: we're using URLs (and if we ever stopped using URLs a lot more would change).  We could use a URI object or some (non-`java.net`) URL object to construct it, or just inline `://` to URL construction.

Reply:
Thanks, I Inlined :// to URL construction